crl-test/service_accounts.tf

```diff
@@ -21,6 +21,14 @@ locals {
     "roles/storage.admin",
     "roles/bigquery.admin",
   ]
+
+  # Roles used to manage projects for integration testing.
+  folder_roles = [
+    "roles/resourcemanager.folderAdmin",
+    "roles/resourcemanager.projectCreator",
+    "roles/resourcemanager.projectDeletor",
+    "roles/resourcemanager.projectIamAdmin",
+  ]
```

Comment:
Optional: I think folderAdmin includes all permissions in projectIamAdmin, but it's fine to keep both

Reply:
thanks, done.